pkg/monitor/monitor.go

```diff
@@ -114,7 +114,8 @@ func (m *Monitor) Stop(ctx context.Context) (ResultState, error) {
 	m.junits = append(m.junits, computedJunit...)
 
 	fmt.Fprintf(os.Stderr, "Evaluating tests.\n")
-	finalEvents := m.recorder.Intervals(m.startTime, m.stopTime)
+	// compute intervals based on *all* the intervals.  Individual monitortests can choose how to restrict themselves.
+	finalEvents := m.recorder.Intervals(time.Time{}, time.Time{})
```

Comment:
this is needed to allow a monitortest to fail for something that is broken during install.